### ISSUE TITLE
How to serialize an array? 

### ISSUE THREAD

**Issue by AlexCppns** [rails-api/active_model_serializers#891]
I am getting the following error:

NoMethodError - undefined method `read_attribute_for_serialization' for #<Array:0x007fd3e1151898>

Ruby code in controller:
```
@results = Record.search(size: 8, query: { match: { name: { query: params[:name], operator: 'and' } } })
@records = @results.records.records << Record.last(5)
 respond_with(@records, each_serializer: RecordSerializer)
```

**Comment by prasadsurase** [comment]
Use ArraySerializer as 
```ruby
ActiveModel::ArraySerializer.new(@records, each_serializer: RecordSerializer).to_json
```

**Comment by ericdfields** [comment]
@prasadsurase that used to be the case in 0.9.3. I'm trying to upgrade to 0.10.0.rc1 and finding that ActiveModel::ArraySerializer doesn't exist anymore…

**Comment by mcmire** [comment]
Looks like it moved to ActiveModel::Serializer::ArraySerializer.

**Comment by AlexCppns** [comment]
I fixed the problem, thanks :).

**Comment by joaomdmoura** [comment]
Tks everyone :smile: you're awesome
It's hard to follow up ever thread :stuck_out_tongue_closed_eyes:
